rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -517,6 +517,10 @@ http {
             allow ::1;
             {{ end }}
             deny all;
+            
+            client_max_body_size                    "0";
```

Comment:
This does not have to be more than what we can store in `configuration_data` which is currently configured to be `5M`. Can we at least set this to `10M` rather than making it unbound.